spec.html

```diff
@@ -366,13 +371,14 @@ <h1>The AsyncContext Constructor</h1>
       </ul>
 
       <emu-clause id="sec-asynccontext">
-        <h1>AsyncContext ( )</h1>
+        <h1>AsyncContext ( _name_ )</h1>
```

Comment:
Sorry, we should bring this up in committee and add it to the spec afterwards. That with the `defaultValue` API.

Reply:
removed and opened an issue to track it: https://github.com/tc39/proposal-async-context/issues/29.